lib/schemas.py

```diff
@@ -16,6 +17,14 @@ class VideoItem(MediaItem):
     folder: Optional[str] = None
     filepath: Optional[str] = None
 
+class YakeItem(GenericItem):
```

Comment:
This appears to be wrong. This should be what is returned after running Yake, i.e., `{"keywords": [["a", 0.98], "b",0.8]]}`